fe/fe-core/src/main/java/org/apache/doris/qe/ConnectContext.java

```diff
@@ -776,6 +794,13 @@ public StmtExecutor getExecutor() {
         return executor;
     }
 
+    public PlSqlOperation getPlsqlQueryExecutor() {
```

Comment:
The method named getPlsqlQueryExecutor but you returned plsql operation

Reply:
done